README.md

```diff
@@ -18,204 +26,191 @@ For more information:
 * [Documentation](https://github.com/krux/postscribe/tree/master/doc)
 * Browse the [raw](https://github.com/krux/postscribe/blob/master/postscribe.js) or [annotated](http://krux.github.com/postscribe/doc/postscribe.html) source code.
 
-
 # Getting Started
 
-PostScribe overrides document.write. It is best and safest to use PostScribe after DOM is ready.
+PostScribe overrides `document.write`. It is best and safest to use PostScribe after DOM is ready.
+
+## Installation
+
+In a browser:
+
+```html
+<script src="./dist/postscribe.min.js"></script>
+```
+
+With an AMD loader:
 
-Include `./htmlParser/htmlParser.js` and `./postscribe.js` on your page. TODO: cdn for postscribe.min.js
+```javascript
+require(['postscribe'], function(postscribe) {});
```

Comment:
A more realistic AMD use case would look like:

```js
define(['postscribe'], function(postscribe) {
  
});
```

Although someone might use postscribe in their main, I think most ppl stick to calling out to other modules.